Resources/Prototypes/_Nyano/Entities/Objects/Devices/pda.yml

```diff
@@ -44,15 +44,25 @@
 
 - type: entity
   parent: BasePDA
-  id: GuardPDA
-  name: guard PDA
+  id: PrisonGuardPDA
```

Comment:
You'll need to revert this change; the PrisonGuardPDA is a DeltaV thing

Reply:
Dont worry im adding it to fix it.